2026/.meta/footer.md

```diff
@@ -6,62 +6,66 @@ is: feedback
 is: social
 ---
 
-# Connect with us on Social
+<div class="sr-only">
```

Comment:
The social media slide image includes the links text on it already, unlike last year (@gavinr-maps intentional?)

Thus, I am keeping the HTML links, but making them screen-reader only for accessibility